packages/pangea-multipass/pyproject.toml

```diff
@@ -1,10 +1,24 @@
-[tool.poetry]
+[project]
 name = "pangea-multipass"
 version = "0.2.0"
 description = "Pangea Multipass authorization library"
-authors = ["Apurv Jawle <apurv.jawle@pangea.cloud>"]
+authors = [
+    { name = "Apurv Jawle", email = "apurv.jawle@pangea.cloud" }
+]
 license = "MIT"
 readme = "README.md"
+requires-python = ">=3.10,<3.13"
+dependencies = [
+    "pangea-sdk",
+    "google-auth-oauthlib",
+    "google-auth-httplib2",
+    "google-api-python-client",
+    "google-auth",
+    "openpyxl",
+    "slack-sdk"
```

Comment:
We should transfer over the below version ranges (those under `tool.poetry.dependencies`), otherwise this is saying "install any version of pangea-sdk, google-auth-oauthlib, etc." See https://github.com/pangeacyber/pangea-python/commit/0492a03e46457d53384e47dccdc8f61d6905c9b6 for how we did it in pangea-python.

Reply:
Fixed. Relative path to `pangea-multipass` still there. It's just for development. It seems that there is no way to use relative path in poetry v2.

Reply:
Yeah the relative path is fine.